node-red-standalone/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: node-red-standalone
+category: Automation
+name: "Node-RED"
+version: "2.2.2-12"
+tagline: Wire together the Internet of Things
+description: >-
```

Comment:
Lets add a note here that this variant of Node-RED has integration with Bitcoin, Lightning and electrs.